test/functions/vass/ignore-unreachable.chpl

```diff
@@ -0,0 +1,70 @@
+// https://github.com/chapel-lang/chapel/issues/20481#issuecomment-1224354640
```

Comment:
Aren't we trying to turn over a new leaf and avoid adding tests to user dirs (or subdirs)? 😄